src/Gold.Redis/Gold.Redis.HighLevelClient/ResponseParsers/JsonResponseParser.cs

```diff
@@ -24,6 +25,12 @@ public T Parse<T>(string response)
             if (response == null)
                 return default(T);
 
+            if (typeof(T) == typeof(string) || typeof(T) == typeof(String))

```

Comment:
Are you sure that both checks are necessary?
maybe 'is' could help here?

Reply:
You are rigth(no need for both string and String check), but we cannot use is operator